layouts/partials/single/content.html

```diff
@@ -14,6 +14,6 @@ <h5 class="subtitle is-5 is-muted">{{ .Params.Subtitle }}</h5>
         <div class="divider"></div>
       </div>
     </div>
-    {{ .Content }}
+    {{ .Content | markdownify }}
```

Comment:
This change breaks rendering on all pages for me, I just get a title and nothing else.

Why is it needed for you @shaloo ?

Reply:
If it's for including the images, I think you want https://gohugo.io/content-management/shortcodes/#figure

Reply:
Will remove that - was experimenting with links in content on a layout/partial/casestudies page. Thx for the heads up.